### PR TITLE
Jargon update: change dev fund to project treasury

### LINKED ISSUE #582: Jargon update: change dev fund to project treasury
As part of the efforts to make Decred more appealing to the general public, we are updating our jargon.

Another proposed change that has broad consensus in the community is changing dev fund to project treasury. 

The rationale is explained here:
https://www.reddit.com/r/decred/comments/95r578/proposed_changes_to_decred_jargon/e3uv2yq

### PR DESCRIPTION
**Pull request by s-ben**
Changes references to 'dev fund' (and other variants, such as 'project fund', etc.) to 'Decred Treasury' ( Closes #582 ). 

### Background

**Please correct if wrong.**

There is broad consensus on changing 'dev fund' (and other variants such as 'project fund') to some variation of [the/Decred/project] Treasury. The most popular choice appears to be 'Decred Treasury'. There was debate whether to put this to a vote on Pi. Some felt having stakeholders vote on this would cause unnecessary voter fatigue. @RichardRed0x 's first proposal proposal on jargon change (which was approved), focused only on more important (and possibly more controversial) terms:
 [Change language: PoS Mining to PoS Voting, Stakepool to Voting Service Provider
](https://proposals.decred.org/proposals/522652954ea7998f3fca95b9c4ca8907820eb785877dcf7fba92307131818c75) 

### Changes

I've replaced all instances of 'dev fund' (and other variants) that I could find with 'Decred Treasury', with the idea that we can use this PR to debate further if we want to go with another term (e.g. [the/Decred/project] Treasury). If we decide to go with another term, it should be easier to do on this branch. 

### Related issue

We do still use the term 'subsidy' in reference to the DRC sent to the Decred Treasury. E.g., on the [proof-of-stake](https://docs.decred.org/mining/proof-of-stake/) page, 

> After a ticket has voted, missed, or expired, the funds (ticket price and **subsidy** if applicable, minus the fee) will enter immature status for another 256 blocks, after which they are released.

I think this is fine, and we can leave as is. Just wanted to toss it out there.

### CODE REVIEW

**Comment by jholdstock** [comment]
Thanks for this Seth. I have never seen anybody object to the term "Treasury" so I think we should just be proactive and get this change done.

I notice that `glossary.md` is included in this PR which I assume is a mistake.

File: docs/getting-started/using-the-block-explorer.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ Option                   | Explanation
 `Timestamp`              | The time this block was created by a miner and was included in the blockchain.
 `Merkle Root`            | A hash value of all the transaction hashes included in this block.
 `Stake Root`             | A hash value of all the stake related transaction hashes in this block. This includes ticket purchases, votes, and ticket revocations.
-`VoteBits`               | (1) Block was approved by proof-of-stake voters. (2) Block was vetoed by proof-of-stake voters and all non-stake transactions in the block were invalidated, along with the proof-of-work miner and development subsidy.
+`VoteBits`               | (1) Block was approved by proof-of-stake voters. (2) Block was vetoed by proof-of-stake voters and all non-stake transactions in the block were invalidated, along with the proof-of-work miner and Decred Treasury.
```
Comment (jholdstock) [inline docs/getting-started/using-the-block-explorer.md]: I think something like the following would be better gramatically: 
```suggestion
`VoteBits`               | (1) Block was approved by proof-of-stake voters. (2) Block was vetoed by proof-of-stake voters and all non-stake transactions in the block were invalidated, along with the newly generated block reward for the proof-of-work miner and the Decred Treasury.
```

File: docs/index.md [hunk 2]
```diff
@@ -10,13 +10,13 @@ Decred is a blockchain-based cryptocurrency with a strong focus on community inp
...
 
 [Proof of Work](mining/proof-of-work.md) miners play a similar role for Decred as they do for Bitcoin, but with Decred they only receive 60% of the block reward.
 
 [Proof of Stake](mining/proof-of-stake.md) voting is central to Decred's governance. Decred holders can time-lock (or "stake") DCR to obtain voting tickets. Tickets are randomly called to vote on-chain; this involves both approving the work of PoW miners and voting Yes/No on any open [rule change proposals](governance/consensus-rules-voting.md). 30% of the block reward goes to the holders of the tickets that voted in that block.
 
-The remaining 10% of the block reward goes into a [Project Treasury fund](https://explorer.dcrdata.org/address/Dcur2mcGjmENx4DhNqDctW5wJCVyT3Qeqkx). Holders of live tickets decide how that treasury is used through [Politeia proposals and voting](governance/politeia.md).
+The remaining 10% of the block reward goes into a [Decred Treasury](https://explorer.dcrdata.org/address/Dcur2mcGjmENx4DhNqDctW5wJCVyT3Qeqkx). Holders of live tickets decide how that treasury is used through [Politeia proposals and voting](governance/politeia.md).
```
Comment (jholdstock) [inline docs/index.md]:
```suggestion
The remaining 10% of the block reward goes into the [Decred Treasury](https://explorer.dcrdata.org/address/Dcur2mcGjmENx4DhNqDctW5wJCVyT3Qeqkx). Holders of live tickets decide how that treasury is used through [Politeia proposals and voting](governance/politeia.md).
```

**Comment by s-ben** [comment]
@jholdstock , have accepted your suggestions and removed glossary; the glossary I think got included because it was open in my text editor, and I corrected a Treasury reference there, adding it to the branch accidentally.

File: docs/governance/introduction-to-decred-governance.md [hunk 3]
```diff
@@ -20,7 +20,7 @@ On-chain voting serves the following purposes:
 
 Decred's on-chain governance is supplemented by *Politeia proposal voting*, which doesn't happen directly on-chain but is woven into the Decred blockchain in some ways. 
 
-Politeia proposals concern the direction of the project, they may involve spending the [project subsidy fund](http://explorer.dcrdata.org/address/Dcur2mcGjmENx4DhNqDctW5wJCVyT3Qeqkx) (10% of the block reward goes into this fund to support development of the project) or amending the [Decred Constitution](decred-constitution.md) or other policies.
+Politeia proposals concern the direction of the project, they may involve the spending of [Decred Treasury funds](http://explorer.dcrdata.org/address/Dcur2mcGjmENx4DhNqDctW5wJCVyT3Qeqkx) (10% of the block reward goes into this fund to support development of the project) or amending the [Decred Constitution](decred-constitution.md) or other policies.
```
Comment (jholdstock) [inline docs/governance/introduction-to-decred-governance.md]:
```suggestion
Politeia proposals concern the direction of the project, they may involve the spending of [Decred Treasury funds](http://explorer.dcrdata.org/address/Dcur2mcGjmENx4DhNqDctW5wJCVyT3Qeqkx) (10% of the block reward goes into the Treasury to support development of the project) or amending the [Decred Constitution](decred-constitution.md) or other policies.
```

**Comment by s-ben** [comment]
@jholdstock , applied your edit using the new 'Apply Suggestion' feature. Looks like it went through, just wanted to give you a heads up in case it didn't go through.

**Pull request merged**